File: Python/tests/test_gng_m7.py
import pytest

from hbayesdm.models import gng_m7


def test_gng_m7():
    _ = gng_m7(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_gng_m7.R
context("Test gng_m7")
library(hBayesDM)

test_that("Test gng_m7", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(gng_m7(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
